cmd/sourced/main.go

```diff
@@ -12,5 +15,25 @@ func main() {
 	composefile.SetVersion(version)
 	cmd.SetVersion(version)
 
+	checkUpdates()
+
 	cmd.Execute()
 }
+
+func checkUpdates() {
+	if version == "master" {
+		return
+	}
+
+	update, latest, err := release.FindUpdates(version)
+	if err != nil {
+		return
+	}
+
+	if update {
+		fmt.Printf(`There is a newer version. Current version: %s, latest version: %s
```

Comment:
could we use the classic yellow for notices?
(I saw that we're not using any logging lib; what if we mark this as `[INFO]`, or something, so it will be easier to find once we start assigning levels to logs?)

Reply:
> once we start assigning levels to logs?

But we don't have any logs. And this is something that I think we did bad in `engine`, the CLI output should not look like a log file in my opinion.

> could we use the classic yellow for notices?

I agree with this. But I'm not sure I want to see log messages for CLI output... I'll investigate if it's better to print a warning somehow formatted to hide the log message look, or use a lib to colorize the output.

Reply:
What about this simple approach in b0dba1e?

![image](https://user-images.githubusercontent.com/1469173/58876642-a2885100-86c6-11e9-9e17-df7990751eec.png)

Reply:
I'm not suggesting with this review, to add a new dep to differentiate errors from info messages, but just walk a bit in that direction; that's because I suggested adding a kind of label in this message.

Anyhow, not really blocker but a small UX improvement; and we can also discuss later about formatting libraries, messages or logs.

Reply:
b0dba1e LGTM and imo it goes in the good direction! :bowing_man: